src/compose/ComposeBox.js

```diff
@@ -279,16 +284,18 @@ export default class ComposeBox extends PureComponent<Props, State> {
 
     return (
       <View>
-        <TopicAutocomplete
-          isFocused={isTopicFocused}
-          text={topic}
-          onAutocomplete={this.handleTopicChange}
-        />
-        <AutoCompleteView
-          text={message}
-          onAutocomplete={this.handleMessageChange}
-          selection={selection}
-        />
+        <View style={[componentStyles.autocompleteWrapper, { marginBottom: totalHeight }]}>
```

Comment:
I tried it to move it to `Popup` as wrapper,  but didn't got success. I tried several other ways too, and now I can guess that `position: 'absolute'` will not work in the child if parent style is not absolute (on Android)